libs/components/src/lib/card/base/index.tsx

```diff
@@ -98,7 +98,7 @@ export const BaseCard: React.FC<BaseCardProps> = ({
               'flex flex-col',
               textColorVariantClass[color],
               sizeVariantTextGap[size],
-              !content && 'flex-1',
+              !content && description && 'flex-1',
```

Comment:
Have to add description in the condition because we don't have description here and it flex grow the div which is not follow the Figma design gap.
<img width="401" alt="image" src="https://github.com/deriv-com/deriv-com-v2/assets/142988136/a82bee79-69b5-42dc-b540-e949ddc66e8a">